sabnzbd/newsunpack.py

```diff
@@ -143,14 +143,13 @@ def check(path, program):
         sabnzbd.newsunpack.PAR2C_COMMAND = sabnzbd.newsunpack.PAR2_COMMAND
 
     if not (sabnzbd.WIN32 or sabnzbd.DARWIN):
-        version, original = unrar_check(sabnzbd.newsunpack.RAR_COMMAND)
+        version, original = sabnzbd.newsunpack.unrar_check(sabnzbd.newsunpack.RAR_COMMAND)
```

Comment:
The sabnzbd.newsunpack prefix probably isn't needed, since you moved the code back.